Gemfile

```diff
@@ -10,9 +10,10 @@ group :development do
 end
 
 gem 'rake'
-gem "jruby-openssl", :platforms => :jruby
+gem 'jruby-openssl', :platforms => :jruby
 gem 'msgpack-jruby', :require => 'msgpack', :platforms => :jruby
 gem 'msgpack', '~> 1.0', :platforms => [:mri, :rbx]
 gem 'bcrypt'
+gem 'openssl', platforms: :mri
```

Comment:
I'm ok to remove Ruby 2.2 from the CI support matrix, but I don't want to break compatibility with older Ruby releases unnecessarily. So can we change this to:

    install_if -> { Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.3.0') } do
      gem 'openssl', platforms: :mri
    end

This will allow the aerospike gem to still be installed on Ruby v2.2 and older. Even TLS encryption might still work since the standard library includes a version of the OpenSSL library anyway. That version probably does not support the latest TLS versions, new ciphers, etc. So upgrading to a later Ruby version is still highly recommended but at least the client is still usable on older Ruby versions.

Reply:
On a similar note. What's your stance on supporting JRuby? I think it can be done without too much effort, but we can address that in a separate PR if you think it's a priority